pyproject.toml

```diff
@@ -154,33 +154,24 @@ test = [
 
 # Development (./CONTRIBUTING.md)
 dev = [
-    "aiohttp-wsgi",
-    "boto3",
+    "aiohttp-debugtoolbar",
+    "diffoscope",
     "djlint",
-    "fakeredis",
-    "gcloud-aio-storage",
-    "google-cloud-logging",
-    "klaus@git+https://github.com/jonashaag/klaus",
-    "loggerhead",
+    "flask",
```

Comment:
Why is flask here?

Reply:
[./py/janitor/git_store.py:    from flask import Flask](https://github.com/jelmer/janitor/blob/eb1138b325cf3bf9069ae9917c2b6f834f7928b2/py/janitor/git_store.py#L184)

Reply:
that should be in the git-store extra then though?

Reply:
Moved!